### ISSUE TITLE
SoundManager's audio does not work with createMediaElementSource when motion sound was played

### ISSUE THREAD

**Issue by hohoaisan** [guansss/pixi-live2d-display#140]
When I use `audio` element in Motion Manager's callback `motionManager.on("motionStart", (group, index, audio) => {})` with `createMediaElementSource` for output the sound to different audio destination sources, the error message was below
```
MediaElementAudioSource outputs zeroes due to CORS access restrictions for [file name specified]
```
Some stackoverflow answers suggested adding crossOrigin attribute with value "anonymous" to audio element, I applied for the build code of [pixi-live2d-display](https://github.com/guansss/pixi-live2d-display) and it works just fine. But not for directly modifying the audio element of motion manager's callback above.

I have to patch the SoundManager for the workaround, but still suggest adding extra `crossOrigin` option than current `volume` for SoundManager

**Comment by guansss** [comment]
Yeah this would be a nice feature. There's already a `crossOrigin` option when creating the model, but that only applies to image elements, I'll make it also work for audios.
